### DISCUSSION TITLE
Draw child item at unzoomed scale

### DISCUSSION THREAD

**Discussion by mgnslndh** [wieslawsoltes/PanAndZoom#73] [Q&A]
How can setup a ZoomBorder child so that it is drawn at the correct location but unscaled?

Let's say for example that I have a control that can draw the origin of the coordinate system. It should always be drawn at the same size independet of the zoom level.

![image](https://github.com/wieslawsoltes/PanAndZoom/assets/1139118/841b67d9-5e36-45ca-bd66-9de788571fb9)

Is there any example of doing this?

**Comment by greenwaym** [comment]
Better late then never, maybe this could help others. I also have controls that should be drawn independent of the zoom level. In your example, you have two things that need changing when zooming, as the zoomborder applies a RenderTransform to the child components: 

I assume you use a custom component with overridden Render function, and draw your shapes with the DrawingContext api:

- you would need to divide your line width with the zoomRatio from the zoomborder. This way, the line width stays constant independent of the zoom value. 
- If you want to keep your geometry the same size, you can use the Geometry.Transform property of your geometry  to counteract the zoomRatio.
